src/commcare_cloud/fab/operations/release.py

```diff
@@ -272,6 +272,15 @@ def update_virtualenv(full_cluster=True):
     """
     roles_to_use = _get_roles(full_cluster)
 
+    def pip_uninstall(cmd_prefix, requirements=(), fail_if_absent=False):
+        assert requirements
+        r_flags = ''
+        for requirements_file in requirements:
+            if not fail_if_absent or files.exists(requirements_file):
+                r_flags += '-r {}'.format(requirements_file)
```

Comment:
will this make something like `-r file1.txt-r file2.txt` (note missing space between `file1.txt` and `-r`) if there are multiple requirements files?

Suggestion: 
```diff
- r_flags = ''
+ r_flags = []
```
and then
```suggestion
                r_flags.extend(['-r', requirements_file])
        r_flags = " ".join(r_flags)
```